README.md

```diff
@@ -19,53 +19,161 @@ how it works in [this blog post](http://blog.humphd.org/thimble-and-bramble/).
 
 Thimble requires a modern web browser, and we recommend using Mozilla Firefox or Google Chrome.
 
-Setup
------
+#Setup/Installation
 
-### Requirements
+Thimble interacts with the Publish API (source managed in [publish.webmaker.org](https://github.com/mozilla/publish.webmaker.org)) to store users, projects, files and other content as well as publish user projects.
 
-- Node.js (version 4.6 or later) [[download](https://nodejs.org/en/download/)]
-- Virtualbox (version 5.1 or later) [[download](https://www.virtualbox.org/wiki/Downloads)]
-- Vagrant (version 1.9 or later) [[download](https://www.vagrantup.com/downloads.html)]
-  - __Note:__ On Windows machines, you may need to restart your computer after installing Vagrant for it to be fully usable
+For authentication and user management, Thimble uses Webmaker OAuth which consists of the Webmaker ID System (source managed in [id.webmaker.org](htps://github.com/mozilla/id.webmaker.org)) and the Webmaker Login API (source managed in [login.webmaker.org](https://github.com/mozilla/login.webmaker.org)).
 
-The setup of Thimble can be divided into two distinct sections:
+All three services along with Thimble are bundled together using Git subtrees and can either be run together using Vagrant, or [manually](#1b-manual-installation).
 
-### Editor
+# 1) Installing Thimble and Services
+##1a) Automated Installation (Preferred Method)
+**Note:** If you aren't able to properly run virtualization software on your machine (for e.g. some versions of Windows only allow one virtualization client to run at a time and if that isn't VirtualBox, you can't run the required VirtualBox as well. This is often a problem if you have docker installed on Windows) or are trying to host Thimble on your own, refer to the [Manual Installation](#1b-manual-installation) instructions instead.
 
-To fully use Thimble locally, you need to first setup Brackets locally first. This can be done by following the steps outlined below:
+### Pre-Requisites for Automated Installation
+In order for Thimble to be installed correctly, the following dependencies needs to be installed in order:
```

Comment:
"following dependencies needs" should be changed to "following dependencies need"